src/Api.elm

```diff
@@ -85,6 +88,15 @@ hentSpråkkoder msgConstructor =
         }
 
 
+leggTilSammendrag : (Result Error Sammendrag -> msg) -> String -> Cmd msg
```

Comment:
Burde vi kanskje endre navn på disse til putSammendrag, post, get osv for å representere hvilken HTTP-request de gjør?

Reply:
+1 (du mener for alle, ikke sant? :) )